check.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+
+try_installing() {
+
+    if [ ! $NO_XBPS ]; then
+        echo "\nTrying to install '$PKG' ... "
+        if [ ! -z $SU_CMD ]; then
+            $SU_CMD xbps-install $PKG
+
+            # 'xbps-install' exits with 0 when the user aborts, so we can't
+            # check return status; use 'which' to check status instead.
+
+            if ( ! which mdbook 2>&- 1>&- ); then
```

Comment:
I think this which is checking for the wrong package, isn't it?

Reply:
No; the *executable* is `mdbook`, even though the *package* is `mdBook`.

Reply:
Oh, sorry, i misunderstood. Fixed.